### PR TITLE
LRDOCS-8561 (LPS-114145): Autosave Fragments as Draft

### PR DESCRIPTION
**Pull request by JamesAGarcia**
This PR contains three commits in fulfillment of [LRDOCS-8561](https://issues.liferay.com/browse/LRDOCS-8561) ([LPS-114145](https://issues.liferay.com/browse/LPS-114145)), with some content that requires SME review.

For SME...
7057656: See the _Using the Fragments Editor_ article, [ln 85](https://github.com/Alec-Shay/liferay-learn/commit/70576561088c6ae6c96cd25796a2e71f52c72e39#diff-404c8c90f98ed1b1c6b536334d648ca493bf97a549aa2a5bb460efc677312cd5R85).

08fa86a: See the _Automatically Importing Fragments_ article, [ln 57](https://github.com/Alec-Shay/liferay-learn/commit/08fa86af8ffbc4764bee72476866c62ca4b09355#diff-f55b7f43447e0bc7f1b30890fe06eeeac4b9e94f69bfbd5b24e397601b5f43d5R57).

7ed4af0: See the _Manage Page Fragments_ article, [ln 25](https://github.com/Alec-Shay/liferay-learn/commit/7ed4af089c2bbdeb496599af13599ad610d84146#diff-d06e1b9bbd48e372ac15d9a63528ec1c5ad4df42bfb1fbf3b14a896c435fe7baR25).

### CODE REVIEW

File: docs/dxp/7.x/en/site-building/developer-guide/developing-page-fragments/auto-deploying-fragments.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ First, deploy an example to see how auto-deployable Fragments work:
 
 1. Verify that the Fragment Collection is available. Open your browser to `https://localhost:8080`, open the Product Menu, and go to Site &rarr; *Site Builder* &rarr; *Page Fragments*. The Collection is listed with the other Collections.
 
+```note::
+   If an imported Fragment has invalid rules, it is automatically saved as a draft.
```
Comment (Alec-Shay) [inline docs/dxp/7.x/en/site-building/developer-guide/developing-page-fragments/auto-deploying-fragments.md:57]: Hey @JamesAGarcia,

I think the changes overall look good, though I'm wondering whether this satisfies with LPS-114145 as a ticket. It's not just when the Fragment has invalid rules that the draft is automatically saved, right?

Comment (JamesAGarcia) [reply docs/dxp/7.x/en/site-building/developer-guide/developing-page-fragments/auto-deploying-fragments.md:57]: Aye, it is automatically saved as a draft when being developed. Though I think all published fragments imported are not saved as drafts, unless they have invalid rules that prevent them from being directly published in the specific Site/instance. But I haven't tested/confirmed this, so I could be mistaken.

Comment (Alec-Shay) [reply docs/dxp/7.x/en/site-building/developer-guide/developing-page-fragments/auto-deploying-fragments.md:57]: Ahh -- so this line is here to clarify that it is saved as a draft as opposed to actually being published? (And the same as the note added to this line, I assume? https://github.com/Alec-Shay/liferay-learn/pull/66/commits/7ed4af089c2bbdeb496599af13599ad610d84146#diff-d06e1b9bbd48e372ac15d9a63528ec1c5ad4df42bfb1fbf3b14a896c435fe7baR25)   

I was under the impression that this note (and possibly also that one other note) was meant to address just the auto-save feature existing in general, which seems a bit more broad.

**Comment by Alec-Shay** [comment]
Thanks! See: https://github.com/sez11a/liferay-learn/pull/285

**Pull request closed without merging**